packages/client/mutations/toasts/mapTeamsLimitExceededToToast.ts

```diff
@@ -0,0 +1,38 @@
+import graphql from 'babel-plugin-relay/macro'
+import {Snack} from '../../components/Snackbar'
+import {OnNextHistoryContext} from '../../types/relayMutations'
+import {mapTeamsLimitExceededToToast_notification} from '../../__generated__/mapTeamsLimitExceededToToast_notification.graphql'
+import SendClientSegmentEventMutation from '../SendClientSegmentEventMutation'
+
+graphql`
+  fragment mapTeamsLimitExceededToToast_notification on NotifyTeamsLimitExceeded {
+    id
+    organization {
+      id
+      name
+    }
+  }
+`
+
+const mapTeamsLimitExceededToToast = (
+  notification: mapTeamsLimitExceededToToast_notification,
+  {history, atmosphere}: OnNextHistoryContext
+): Snack => {
+  const {id: notificationId, organization} = notification
+  const {name: orgName} = organization
+
+  return {
+    autoDismiss: 0,
+    key: `newNotification:${notificationId}`,
+    message: `Your account is on a roll! Check out "${orgName}"'s usage`,
+    action: {
+      label: 'See Usage',
+      callback: () => {
+        history.push(`/usage`)
+        SendClientSegmentEventMutation(atmosphere, 'Clicked usage snackbar CTA')
```

Comment:
+1 I'd like to refactor this event to `Feature CTA Clicked`, with `feature` = "Usage" and `featureCtaLocation` = "Snackbar", but know that it is in use elsewhere in the codebase.

If we decide to merge this as is (which we should do unless we refactor everywhere), I'll open a new issue refactor similar events after I finalize our Segment event naming policy.

Reply:
@tghanken I suggest to create a new refactoring issue, to keep this PR small